hyperspy/_signals/signal1d.py

```diff
@@ -1036,8 +1036,8 @@ def _remove_background_cli(
         if fast and not self._lazy:
             try:
                 axis = self.axes_manager.signal_axes[0].axis
-                result = Signal1D(self.data -
-                                  background_estimator.function_nd(axis))
+                result = self - Signal1D(
+                        background_estimator.function_nd(axis))
```

Comment:
Actually, `Signal1D` is not necessary here and by removing it we can avoid the overhead of the creation of a signal, which make it 10% faster, not very significant but still something.

Reply:
Good point! Fixed.